src/converter/modules/ExlClient.js

```diff
@@ -93,6 +95,53 @@ export default class ExlClient {
     throw new Error(`Playlist with id: ${id} not found`);
   }
 
+  /**
+   * Get non-English labels from EXL Config populated endpoints
+   * @param {string} endpoint
+   * @param {string} id
+   * @param {string} lang
+   * @returns {string}
+   */
+  async getLabelFromEndpoint(endpoint, id, lang = 'en') {
```

Comment:
a few things:

1. The response should be cached 24 hours, see `getSolutions` for an example of how we've done that.
2. The API has 2 distinct endpoints: /api/features and /api/roles, I'd expect there be two methods in the client: `getFeatures` and `getRoles` and that responses are cached.
3. If language is `en` wouldn't we want to avoid this all together?

Reply:
1. Added caching
2. There are 5 apis with an identical format include the roles, levels, and features we're using here. I added a constant with them at the top with the expectation that folks would use the values in it with this function.
3. Added a condition for this.